src/sentry/lang/javascript/processor.py

```diff
@@ -184,7 +184,7 @@ def discover_sourcemap(result):
 
 
 def fetch_release_file(filename, release):
-    cache_key = 'releasefile:1:%s:%s' % (
+    cache_key = 'releasefile:v1:%s:%s' % (
```

Comment:
Just made this version name consistent with what we do for the `source:cache:` keys.